cluster/juju/layers/kubernetes-worker/sos-scripts/kubectl

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -ux
+
+sudo -iu ubuntu kubectl cluster-info > $SOS_SCRIPT_DIR/cluster-info
+sudo -iu ubuntu kubectl cluster-info dump > $SOS_SCRIPT_DIR/cluster-info-dump
+for obj in pods svc ingress secrets pv pvc rc; do
+  sudo -iu ubuntu kubectl describe $obj --all-namespaces > $SOS_SCRIPT_DIR/describe-$obj
```

Comment:
Nope, cleared that up :)  I wasn't comprehending what was happening in that block and this resolves my concern. 👍